vllm/v1/core/sched/scheduler.py

```diff
@@ -729,16 +728,11 @@ def _make_cached_request_data(
                     req.num_computed_tokens : req.num_computed_tokens + num_tokens
                 ]
                 new_token_ids.append(token_ids)
-            elif use_connector:
-                # When using a KVConnector, we add a placeholder to avoid index
-                # out of bounds errors. TODO: Remove this once the KVConnector
-                # is updated to handle token IDs properly.
-                new_token_ids.append([])
```

Comment:
Not directly related, but no longer needed.

Reply:
@njhill Can we make a separate PR for this?